ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -2,7 +2,7 @@
   class="flight-icon flight-icon-{{@name}} {{if this.display "flight-icon-display-inline"}}"
   ...attributes
   aria-hidden="true"
-  data-test-icon
+  data-test-icon={{this.iconId}}
```

Comment:
thought: maybe we can double check wit the teams using this attribute(s) what they expect this value to be. (I for example would expect to have the icon name as value; in my previous company these `data-test` attributes were using in automation testing to select a DOM element or check their existence, so having a random value would not work, but maybe here is different). 

suggestion: we could even discuss if they want/need more than one attribute (in that case expose a `data-test` as object of keys/values)

Reply:
@dhaulagiri if I change it to name, will it cause issues if it's a test for a page with two of the same icons on it?

Reply:
It could, but my expectation would be that the developer should have something local to that icon invocation that they can use to scope the selector more narrowly. Or they would be testing a component in isolation that is less likely to have multiple instances of a single icon. 

This is where @didoo's suggestion of taking a look at some existing codebases would be helpful as I'm pretty sure the structure component is already doing something similar.

Either way, I'm pretty sure that having it be based on the id would cause this value to be non-static which seems problematic for test scenarios.